### PR TITLE
feature: ZENKO-3163 Update Zenko Version CRs

### PR DESCRIPTION
**Pull request by miniscruff**
* Update dependency versions
* Update operator CRDs

### CODE REVIEW

File: eve/workers/end2end/scripts/deploy-zenko.sh [hunk 1]
```diff
@@ -66,10 +66,10 @@ spec:
     management:
       ui:
         image: registry.scality.com/sf-eng/zenko-ui
-        tag: 'c091051'
+        tag: '0.1.0'
```
Comment (tcarmet) [inline eve/workers/end2end/scripts/deploy-zenko.sh]: Here I think it's `image: registry.scality.com/zenko-ui/zenko-ui`

File: eve/workers/end2end/scripts/deploy-zenko.sh [hunk 2]
```diff
@@ -65,11 +65,11 @@ spec:
   versions:
     management:
```
Comment (anurag4DSB) [inline eve/workers/end2end/scripts/deploy-zenko.sh]: Do we need to add vault in this file as well?
I do not know how this works yet. But still checking

Comment (miniscruff) [reply eve/workers/end2end/scripts/deploy-zenko.sh]: Vault is optional and has to be enabled which is not the case in the zenko CR below. Now if you think we should be using it here that is a different question that I am not really the one to answer :|

Comment (anurag4DSB) [reply eve/workers/end2end/scripts/deploy-zenko.sh]: I might not have the complete context, clarifying just to learn more.
Are you saying Vault is optional in Zenko 2.0.0-alpha.X?
or 
Vault is optional in end2end tests?

Comment (miniscruff) [reply eve/workers/end2end/scripts/deploy-zenko.sh]: > Are you saying Vault is optional in Zenko 2.0.0-alpha.X?

Currently yes, it must be enabled through the zenko CR.

> Vault is optional in end2end tests?

Also yes as currently, it seems to be disabled.

**Comment by alexandre-merle** [comment]
should we wait for https://github.com/scality/cloudserver/pull/3312 as well ?

**Comment by miniscruff** [comment]
> should we wait for [scality/cloudserver#3312](https://github.com/scality/cloudserver/pull/3312) as well ?

I am not sure I can decide if we should, it sounds like a good idea to me and only takes a few minutes to update here once the tag is made.

**Comment by alexandre-merle** [comment]
It's merged, so if you want to update the tag, you can :)

**Comment by miniscruff** [comment]
> It's merged, so if you want to update the tag, you can :)

Is there a new release or just a git hash? We are trying to use tags when they exist. ( a few projects don't have any tags yet )

File: solution/zenkoversion.yaml [hunk 3]
```diff
@@ -33,25 +33,32 @@ spec:
...
-      tag: '1.2'
+      tag: '1.8.2'
     zookeeper:
       image: zookeeper
       tag: 0.2.4
     kafka:
-      image: kafka
-      tag: 2.12-2.3.0
+      cluster:
+        image: wurstmeister/kafka
```
Comment (alexanderchan-scality) [inline solution/zenkoversion.yaml]:
```suggestion
        image: kafka
```

**Review by alexanderchan-scality — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**